diff --git a/node_modules/@wordpress/env/lib/runtime/docker/docker-config.js b/node_modules/@wordpress/env/lib/runtime/docker/docker-config.js
index 0820bcc..fd22583 100644
--- a/node_modules/@wordpress/env/lib/runtime/docker/docker-config.js
+++ b/node_modules/@wordpress/env/lib/runtime/docker/docker-config.js
@@ -229,6 +229,8 @@ RUN rm /tmp/composer-setup.php`;
 	dockerFileContent += `
 USER $HOST_UID:$HOST_GID
 ENV PATH="\${PATH}:/home/$HOST_USERNAME/.composer/vendor/bin"
+RUN composer global config --no-plugins audit.abandoned ignore
+RUN composer global config --no-plugins audit.block-insecure false
 RUN composer global require --dev phpunit/phpunit:"^5.7.21 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0"
 USER root`;
 
